doc/usage/tutorial.md

```diff
@@ -49,7 +49,7 @@ spec:
             spec:
               containers:
                - name: stateful-set-2-replicas
-                 image: k8s.gcr.io/echoserver:1.4
+                 image: k8s.gcr.io/echoserver:1.4 # for Mac M1 use kicbase/echo-server:1.0
```

Comment:
@sunya-ch I really appreciate your contribution to fixing this document. Would it be possible update the image name to the kicbase image? The e2e tests use it are known to work on MI Machine.